database/things/php.pldb

```diff
@@ -3,6 +3,7 @@ appeared 1995
 type pl
 creators Rasmus Lerdorf
 website https://php.net
+documentation https://devdocs.io/php/
```

Comment:
we should also add https://www.php.net/docs.php. `documentation` keyword should allow multiple, if we don't support that yet.

Reply:
Will definitely check that.